src/user/configure-metadata.md

```diff
@@ -4571,6 +4574,8 @@ Table: External map layer objects in the Maintenance app
         be added from the Add Layer selection and placed anywhere above
         the base map.
 
+    Note that Vector Style layers can only be added as a Basemap.
```

Comment:
"basemap" in lowercase. We should also change from "base map" to "basemap" in line 4575.